etc/helm/pachyderm/values.yaml

```diff
@@ -420,11 +420,17 @@ pachd:
       id: ""
       secret: ""
     minio:
+      # minio bucket name
       bucket: ""
+      # the minio endpoint. Should only be the hostname:port, no http/https.
       endpoint: ""
+      # the username/id with readwrite access to the bucket.
       id: ""
+      # the secret/password of the user with readwrite access to the bucket.
       secret: ""
+      # enable https for minio with "true" defaults to "false"
       secure: ""
```

Comment:
I'm not really an authority on the helm chart. But IMO this should be a boolean and not a string.

@chainlink @acohen4 any thoughts?

Reply:
I agree it should be a boolean. Only concern is helm backwards compatibility - we usually don't want to modify existing field types, especially not in point releases

Reply:
Yeah thats why I left it as a string, especially because `ParseBool` can handle both options cleanly